azure-stack/user/azure-stack-solution-template-kubernetes-dashboard.md

```diff
@@ -122,6 +122,14 @@ You can use the dashboard. For more information on the Kubernetes dashboard, see
 
 ![Azure Stack Hub Kubernetes Dashboard](media/azure-stack-solution-template-kubernetes-dashboard/azure-stack-kub-dashboard.png)
 
+## Troubleshooting
+
+## Custom Virtual Networks
+
+If you face connectivity issues accessing the Kubernetes dashboard after you deploy Kubernetes to a [custom virtual network](https://docs.microsoft.com/en-us/azure-stack/user/kubernetes-aks-engine-custom-vnet), ensure that the target subnet/s are linked to the route table and network security group resources created by AKS engine.
```

Comment:
Can you change the URL (remove the locale): 

https://docs.microsoft.com/en-us/azure-stack/user/kubernetes-aks-engine-custom-vnet
to:
https://docs.microsoft.com/azure-stack/user/kubernetes-aks-engine-custom-vnet